fontes/avaliador-sintatico/dialetos/avaliador-sintatico-pitugues.ts

```diff
@@ -889,7 +885,7 @@ export class AvaliadorSintaticoPitugues
                 espacosIndentacaoLinhaAtual = this.pragmas[simboloAtual.linha].espacosIndentacao;
             }
         }
-
+        this.analisarTextoDeDocumentacao()
```

Comment:
O retorno deve ser adicionado a `declaracoes`. 

```suggestion
        declaracoes.push(this.analisarTextoDeDocumentacao());
```